common/ts/prepare-task.ts

```diff
@@ -46,6 +46,10 @@ export default async function prepareTask(endpoint: Endpoint, rootPath: string)
     })
   );
 
+  if (scannerMode === ScannerMode.CLI) {
+    tl.setVariable('SONAR_SCANNER_OPTS', scanner.toCliProps());
```

Comment:
I'm really surprised by this change. SONAR_SCANNER_OPTS is intended to pass JVM arguments, not scanner properties. What if users already try to set SONAR_SCANNER_OPTS to increase memory (like -Xmx)?